openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -0,0 +1,245 @@
+            ["Salaires bruts",null, results.salaires_de_base, results.salaires_de_base],
+            ["Cotisations salariales", "Salaires bruts", results.cotisations_salariales, - results.cotisations_salariales],
+
+            ["Impots directs", "Salaires bruts", results.impots_directs , - results.impots_directs],
+            ["Revenu disponible", "Salaires bruts", results.revenu_disponible, results.revenu_disponible],
+            ["Revenu du travail", "Revenu disponible", results.revenus_du_travail, results.revenus_du_travail],
+            ["Prestations sociales", "Revenu disponible", results.prestations_sociales, results.prestations_sociales],
+          ]);
+
+          tree = new google.visualization.TreeMap(document.getElementById("chart_div"));
+
+          tree.draw(data, {
+            minColor: "#5bc0de",
+            midColor: "#5bc0de",
+            maxColor: "#5cb85c",
+            headerHeight: 55,
+            fontColor: "black",
+            showScale: true
+          });
+
+        }
+        document.getElementById("graph").innerHTML="<div id='chart_div' style='width: 100%; height: 400px;'></div>";
+
+      }
+  </script>
+</%def>
+
+<%def name="h1_content()" filter="trim">
+Calcul du revenu disponible
+</%def>
+
+
+<%def name="page_content()" filter="trim">
+      <p class="lead">Le revenu disponible représente l'ensembles des revenus issus du travail, des pensions, des revenus du capital et des prestations sociales, auxquelles on soustrait les impots directs du ménage.</p>
+      <!-- Data input -->
+      <div class="row">
+        <div class="col-md-4">
+          <form action="javascript:void(0);">
+            <p><b>Salaire annuel brut du premier conjoint :</b></p>
+            <p>
+              <input type="text" onkeyup=updateSalaries("salary1",this.value) size="20" value="25000"> €/an
+            </p>
+            <p><b>Salaire annuel brut du second conjoint :</b></p>
+            <p>
+              <input type="text" onkeyup=updateSalaries("salary2",this.value) size="20" value="15000"> €/an
+            </p>
+          </form>
+        </div>
+        <div class="col-md-8">
+          <p>Ce ménage, composé de 2 adultes sans enfants, et qui ne perçoit ni pensions ni revenus du capital :</p>
+          <span id="situationDescription"></span>
+        </div>
+      </div>
+      <hr>
+      <!-- Visualisation -->
+      <div class="row">
+        <div class="col-md-4">
+          <h3>Visualisation</h3>
+          <ul>
+            <li>Cliquez sur "Revenu disponible" pour en voir le détail.</li>
+            <li>Faites un clic-droit pour revenir à la distribution précédente.</li>
+          </ul>
+        </div>
+        <div class="col-md-8">
+          <span id="graph"></span>
+        </div>
+      </div>
+      <!-- Version d'OpenFisca -->
+      <div class="row">
+        <div class="col-md-12">
+          <hr>
+          <p>OpenFisca-France@18.6</p>
```

Comment:
That could be a big and mean lie. if we're not able to get the version number right now, just don't mention the version number ;)